lib/pact_broker/api/renderers/html_pact_renderer.rb

```diff
@@ -57,8 +57,10 @@ def pact_metadata
                 <span class='value'>#{@pact.consumer_version_number}#{tags}</span>
               </li>
               <li>
-                <span class='name'>Date published:</span>
-                <span class='value'>#{published_date}</span>
+                <span class='name' title='#{published_date}'>Date published:</span>
+                <span class='value' title='#{published_date}'>
+                  #{PactBroker::DateHelper.distance_of_time_in_words(@pact.created_at.to_time, DateTime.now)} ago
+                </span>
```

Comment:
Can you refactor `PactBroker::DateHelper.distance_of_time_in_words(@pact.created_at.to_time, DateTime.now)` into its own method like the other lines please.